src/main/java/uk/gov/hmcts/reform/managecase/api/controller/NoticeOfChangeController.java

```diff
@@ -42,23 +55,34 @@
     @SuppressWarnings({"squid:S1075"})
     public static final String GET_NOC_QUESTIONS = "/noc-questions";
     public static final String VERIFY_NOC_ANSWERS = "/verify-noc-answers";
+    public static final String REQUEST_NOTICE_OF_CHANGE_PATH = "/noc-requests";
+    public static final String CHECK_NOTICE_OF_CHANGE_APPROVAL_PATH = "/check-noc-approval";
 
     public static final String VERIFY_NOC_ANSWERS_MESSAGE = "Notice of Change answers verified successfully";
-
-    public static final String REQUEST_NOTICE_OF_CHANGE_PATH = "/noc-requests";
     public static final String REQUEST_NOTICE_OF_CHANGE_STATUS_MESSAGE =
         "The Notice of Change request has been successfully submitted.";
+    public static final String APPROVAL_IS_NOT_CONFIGURED_IN_THE_CASE = "auto approval is not configured for the case";
+    public static final String CHECK_NOC_APPROVAL_DONE = "check noc approval has done";
```

Comment:
`Check Notice of Change approval has completed`

or

`Check Notice of Change approval has finished`

Reply:
Alternative wording now applied in as an update for https://github.com/hmcts/aac-manage-case-assignment/pull/129#discussion_r521519161